convention.md

```diff
@@ -3,10 +3,35 @@
 Version: **<!--VERSION-->x.x.x<!--VERSION-->**
 Date: **<!--DATE-->01. Jan 2000<!--DATE-->**
 
+## Introduction
+
+​The Homie convention is an open standard framework designed to facilitate the
+communication and integration of Internet of Things (IoT) devices using the
+[MQTT protocol](https://mqtt.org).
+
+In particular, the Homie convention defines a consistent topic structure and
+messaging format that enables devices to represent themselves, their data and
+their supported controls/commands in a uniform manner, enabling automatic
+discovery.
+
+## Definitions
```

Comment:
Wouldn't "roles" be a better description? Because there should be way more entries here if we call it definitions.